### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by bitsurgeon**
- update readme
- use ssd mobilenet v1 for simulator

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,114 +1,144 @@
...
 
-* Training the model 
+* Labelling Images:
+  * We used [LabelImg](https://github.com/tzutalin/labelImg) to label the collected images.
 
+* Create TFRecord File:
+  * To use our own dataset in Tensorflow Object Detection API, we must convert it into the [TFRecord](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/using_your_own_dataset.md) file format.
 
+* Training Model:
+  * Our models get trained on both [AWS](https://aws.amazon.com/) and [GCP](https://cloud.google.com/) to speed up the overall process.
```
Comment (yz540) [inline README.md]: Add one more step about export model for tf 1.4 to run on Carla?

File: README.md [hunk 2]
```diff
@@ -1,114 +1,144 @@
...
-The images from simulator and test site are very different. Our traffic classification use one model for the simulator and one model for the test site. The training of the model includes three steps:
-* Collecting images:
-  * simulator: use the state of the /vehicle/traffic_lights in the simulator to get the ground truth images.
-  * site: the images are from the rosbag.
+We have learned a lot about how to prepare data and training model from [Tensorflow Object Detection API](https://github.com/tensorflow/models/tree/master/research/object_detection) and got inspired from these past Udacity fellow students:
+- [Jose Horas](https://github.com/josehoras/Self-Driving-Car-Nanodegree-Capstone)
+- [Marco Marasca](https://github.com/marcomarasca/SDCND-Traffic-Light-Detection)
 
-* Labelling images: We used [LabelImg](https://github.com/tzutalin/labelImg) to label the collected images.
+The images from simulator and test site are very different in terms of shape and position. Our traffic classification use one model for the simulator and one model for the test site. The training of the model includes three steps:
```
Comment (yz540) [inline README.md]: 5 steps including the exporting model

File: README.md (hunk 1 above)
Comment (bitsurgeon) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (bitsurgeon) [reply README.md]: done

**Comment by bitsurgeon** [comment]
since there is one member reviewed, and I will just go ahead with the release according to our plan, thanks

**Pull request merged**